docs/developers/node-tutorial.mdx

```diff
@@ -735,15 +735,15 @@ With your wallet funded, you can move on to the next step.
 ## Node API Calls
 
 Open up another terminal window in order to begin querying the API.
-`celestia-node` exposes its RPC endpoint on port `26658` by default.
+`celestia-node` exposes its REST gateway on port `26659` by default.
```

Comment:
We still have RPC endpoint. But I have removed it as we are now referring to REST Gateway for the tutorial below

Reply:
before merging, would like a double-check that this doesn't need to happen elsewhere in docs